src/client/app/components/maps/MapViewComponent.tsx

```diff
@@ -39,7 +41,9 @@ class MapViewComponent extends React.Component<MapViewPropsWithIntl, MapViewStat
 			nameFocus: false,
 			nameInput: this.props.map.name,
 			noteFocus: false,
-			noteInput: (this.props.map.note) ? this.props.map.note : ''
+			noteInput: (this.props.map.note) ? this.props.map.note : '',
+			circleFocus: false,
+			circleInput: (this.props.map.circleSize.toString()) ? this.props.map.circleSize.toString() : '0.15'
```

Comment:
I seems legacy systems (and maybe the current system) can have max_circle_size_fraction be null in the database. (It might be nice to trace down why at some point.) When this happens, this line of code causes an exception and nothing is displayed. I fixed this by changing to:

circleInput: (this.props.map.circleSize !== null && this.props.map.circleSize.toString()) ? this.props.map.circleSize.toString() : '0.15'

Also, in src/server/models/Map.js for mapRow, I had to make it return the same value so the mapping worked with the default (lines 80-1):

return new Map(row.id, row.name, row.displayable, row.note, row.filename, 
row.modified_date, row.origin, row.opposite, row.map_source, row.north_angle, (row.max_circle_size_fraction !== null ? row.max_circle_size_fraction : 0.15));

This all seemed to work fine. Adding comments about why the null check is needed would be good. It may be another change is better to address this and it seems unfortunate to have it in two places.

I considered the alternative of doing a DB migration but that seemed more painful.

Reply:
added null test, and modified Map.js to what was mentioned

Reply:
I just pushed some code to force the route to save the default value into the DB.